### ISSUE TITLE
XCCDF/OVAL Expressed as NIST Metaschema

### ISSUE THREAD

**Issue by djhaynes** [scapcommunity/XCCDF#9]
**Challenge:** There have been discussions that XML is too difficult, too verbose, and that XCCDF/OVAL should migrate to other formats such as YAML or JSON. Redesigning XCCDF/OVAL to use a new format would be a large undertaking, but if there is enough support for it, changing the format could improve the content authoring experience.

**Proposal:** Look into the NIST Metaschema format that was created as part of the OSCAL project as a possible avenue to support new data formats such as JSON.

**Comment by djhaynes** [comment]
NIST Metaschema
https://github.com/usnistgov/OSCAL/tree/master/src/metaschema

**Comment by DavidRies** [comment]
I agree with the challenge/problem, but--at first glance--this approach doesn't sound promising to me. If we translate the current schemas into other formats (potentially via metaschema), won't we have replicated the current complexity? I'm very open to viable simplification solutions that retain the core benefits of SCAP, but currently the authoring tools development initiatives seem the most promising way to address this IMO.

**Comment by djhaynes** [comment]
The group agreed that this issue should be backlogged.
